### PR TITLE
add solution in kotlin for day 1 + update readme

### PR DESCRIPTION
**Pull request by kxmpxtxnt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Hier ist eine Liste mit allen von Beitragenden verwendeten Programmiersprachen u
 
 Here is a list of all programming languages used by the contributors and the days that were solved with them:
 
-| Day/Lang                     | Java | Python | Kotlin | Rust |
-|------------------------------|------|--------|--------|------|
-| 01 - ???        | ❌    |  ❌ | ❌ | ❌ |
```
Comment (dlsf) [inline README.md:15]: Revert this change, the README will be managed by the moderation team

**Review by dlsf — CHANGES_REQUESTED** [review]

**Comment by kxmpxtxnt** [comment]
shit
